ytfzf

```diff
@@ -104,8 +97,9 @@ function_exists "search_prompt_menu" || search_prompt_menu () {
 	if [ $is_ext_menu -eq 1 ]; then
 		search="$(printf '' | external_menu)"
 	else
-		printf "Search: "
+		printf "Search\n> "
 		read -r search
+		printf '\e[1A\e[K\r%s\n' "> $search"
```

Comment:
\e isn't really standard, use \033 instead, also can you like explain what this those escape sequences do.

Reply:
Those escape sequences erase the previous line and return the cursor to the beginning of the line.

`printf '\033[1A\e[K\r%s\n' "> $search"` will erase the previous line and print '> search-query'

1. This is the output if the search query is given as an argument.
```shell
$ ytfzf 'Search query'
Scraping YouTube
Fetching thumbnails...
```

2. This is the output when you are prompted to type the search query after invoking ytfzf.
```shell
$ ytfzf
Search
> 
```
```shell
$ ytfzf
Search
> Search query
Scraping YouTube
Fetching thumbnails...
```

3. This is the output when the search query is piped to ytfzf. This is why I changed the search prompt have 2 lines because if the search query is piped to stdin, then the shell prompt gets deleted if there is only a one line prompt.
```shell
$ echo 'Search query' | ytfzf
> Search query
Scraping YouTube
Fetching thumbnails...
```
`echo 'Search query' | ytfzf -` also works still.